CONTRIBUTORS.md

```
@@ -0,0 +1,23 @@
+| [Claude (Anthropic)](https://claude.ai)                | AI Pair Programmer  | Code, tests, refactors, documentation, code review |
+| [Codex (OpenAI)](https://openai.com/blog/openai-codex) | AI Pair Programmer  | Code, documentation, code review                   |
+
+## How to Contribute
+
+See [CONTRIBUTING.md](CONTRIBUTING.md) for development setup, coding standards, and pull request guidelines.
+
+## Recognition
+
+Human contributors are tracked via [all-contributors](https://allcontributors.org/). To add yourself, comment on any issue or PR with:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)